### PR TITLE
Adds autopsy report because I didn't realize there already was one.

### PR DESCRIPTION
**Pull request by ghost**
For #9 even though as I was committing it all I realized one already existed. Go me, A+ observation skills.

### CODE REVIEW

File: templates/index.json [hunk 1]
```diff
@@ -62,14 +63,14 @@
...
         "Patrol Assignment": "security/patrol_assignment.txt",
-        "Patrol Assignment (Alternate)": "security/patrol_assignment2.txt",
+		"Patrol Assignment (Alternate)": "security/patrol_assignment2.txt",
         "POI Watchlist": "security/personnel_watchlist.txt",
-        "POI Watchlist (Alternate)": "security/personnel_watchlist2.txt",
-        "SECARM Inventory": "security/secarm_inventory.txt",
-        "Shift Report": "security/shift_report.txt"
+		"POI Watchlist (Alternate)": "security/personnel_watchlist2.txt",
+		"SECARM Inventory": "security/secarm_inventory.txt",
+		"Shift Report": "security/shift_report.txt"
```
Comment (Dibasic) [inline templates/index.json]: tabs in the JSON again, looks like you did this from your prev fork

File: templates/medical/autopsy2.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+[hr]
+[center]REVIEW INFORMATION:[/center]
+Vessel/Station Performed: [field]
+
+[small][i]I hereby declare that after receiving notice of the death described herein, I took charge of the body and made inquiries regarding the cause of death in accordance with local laws and standing orders, and that the information contained herein regarding said death is true and correct to the best of my knowledge and belief.[/i]
+[/small]
+Name: [field]
+Signature: [field]
+Organization: [field]
+Rank: [field]
```
Comment (Dibasic) [inline templates/medical/autopsy2.txt]: medical contractors might do this form too, not sure whether having rank is necessary

**Review by Dibasic — CHANGES_REQUESTED** [review]

File: templates/medical/autopsy2.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+[field]
+[hr]
+[center]REVIEW INFORMATION:[/center]
+Vessel/Station Performed: [field]
+
+[small][i]I hereby declare that after receiving notice of the death described herein, I took charge of the body and made inquiries regarding the cause of death in accordance with local laws and standing orders, and that the information contained herein regarding said death is true and correct to the best of my knowledge and belief.[/i]
+[/small]
+Name: [field]
+Signature: [field]
+Organization: [field]
```
Comment (Dibasic) [inline templates/medical/autopsy2.txt:35]:
```suggestion
Organization: [field] Rank (if applicable): [field]
```
Maybe something like this, or put rank by name, to make it work if a civilian does this

File: templates/index.json (hunk 1 above)
Comment (ghost) [reply templates/index.json]: I'll be honest - No clue what's going on with that.

**Review by Dibasic — APPROVED** [review]

**Pull request merged**